app/components/autocomplete/slash_suggestion/slash_suggestion.tsx

```diff
@@ -140,15 +190,16 @@ export default class SlashSuggestion extends PureComponent {
                 Suggestion: '/' + item.trigger,
                 Hint: item.auto_complete_hint,
                 Description: item.auto_complete_desc,
+                IconData: item.icon_url,
```

Comment:
It's not being used, but it's required for this `AutocompleteSuggestion` type to be complete. The slash icon is used unconditionally (see below), even though the server is returning icon URLs for command autocomplete. We either need to show these, or make it so the server does not send back the icon URLs for mobile because they are often data URLs that hold a large blob of base64 data, or we should have some sort of protocol that says that the mobile app already has the icon data so don't send it.

https://github.com/mattermost/mattermost-mobile/blob/5f8a28cbd8a7cf9b5ee63f80a53c30f8aeeddc37/app/components/autocomplete/slash_suggestion/slash_suggestion_item.tsx#L12
https://github.com/mattermost/mattermost-mobile/blob/5f8a28cbd8a7cf9b5ee63f80a53c30f8aeeddc37/app/components/autocomplete/slash_suggestion/slash_suggestion_item.tsx#L100-L105

There's a related ticket here https://mattermost.atlassian.net/browse/MM-33599

Reply:
The size of the base64 strung shouldn't really be a problem unless the user is in a very low bandwidth network. As lf having that huge base64 strkng in the redux store can be problem for performance specially if not used, maybe as it is not used, for now we could ommit its data in the reducer? Or does the server expect to receive it back from the client?

Reply:
The icon data is delivered to the client on every key stroke past the base command. It just seems like a lot of data being transferred, relative to the size of the rest of the command autocomplete suggestion response, and it's not even being used.

> for now we could ommit its data in the reducer?

That sounds fine

> does the server expect to receive it back from the client?

No the autocomplete suggestion structure never gets sent back to the server

Reply:
@enahum Curious what you think about this because we have efforts to limit network traffic for mobile such as base command caching, but are at the same time returning unused image binary data.

Reply:
@mickmister if unused and we can avoid transmiting that over the wire is possible we should